src/itdelatrisu/opsu/options/Options.java

```diff
@@ -399,15 +399,17 @@ public void selectItem(int index, GameContainer container) {
 
 			@Override
 			public String getValueString() {
-				return String.format((getTargetFPS() == 60) ? "%dfps (vsync)" : "%dfps", getTargetFPS());
+				return String.format((getTargetFPS() == -1) ? "Unlimited" : 
+					(getTargetFPS() == 60) ? "%dfps (vsync)" : "%dfps", getTargetFPS());
 			}
 
 			@Override
 			public Object[] getItemList() {
 				if (itemList == null) {
 					itemList = new String[targetFPS.length];
 					for (int i = 0; i < targetFPS.length; i++)
-						itemList[i] = String.format((targetFPS[i] == 60) ? "%dfps (vsync)" : "%dfps", targetFPS[i]);
+						itemList[i] = String.format((targetFPS[i] == -1) ? "Unlimited" : 
```

Comment:
(same as above)

Reply:
Wait, what?

The shortcut function to "Unlimited" has already been removed. Removing this would result in the option "Unlimited" being ***not displayed on the option's dropdown list***, but available via modifying the configuration file's `FrameSync` parameter to -1.